zulipterminal/ui_tools/buttons.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         prefix_character: Union[str, Tuple[Any, str]]='\N{BULLET}',
         text_color: Optional[str]=None,
         count: int=0,
+        count_theme: str='unread_count',
```

Comment:
Would it be better to use `count_color`, to have a name that is consistent with `text_color`?
Also, you might want to consider having a `TypeAnnotation` similar to `text_color`.

Reply:
@Rohitth007 and I discussed naming it count_theme because 'starred_count'/'unread_count' are styles from the theme palette so what's being stored in the arg is the item's style name versus the actual 'color'. Does that make sense? I'd be happy to discuss further and change it if we decide count_color would be better. Good catch with the TypeAnnotation!

Reply:
Actually even `text_color` is a style in the palette. Eg: The user button uses `user_idle`, `user_active` etc.
Actually now that you brought this up I feel both the names feel wrong. 
* `text_color` technically can be used not just to change the text color (foreground) but also the background along with styles like _bold_, _italics_, etc. This can be used when we select a different background color from the default.
* `count_theme` feels wrong as theme is more like a collection of styles?

`text_style` and `count_style` seem apt here. This has to be done in a different commit if going forward with it.
Thoughts?

Reply:
> Now that you brought this up I feel both the names feel wrong.

Right. I was about to say this.
`*_style` here seems fine. `urwid`, however, likes to call those `display_attributes`, but that seems a bit too long IMO?

Re `TypeAnnotation`, Note that there's also a `None` display_attr in `themes`, which could be passed here, so its better to generalize the type as `Optional[str]`.

Reply:
So shall I update the name of count_theme in the original commit? Seems logical to name it correct from the start. I can create a third commit to fix the `text_color` name. I agree with the name change I think it makes a lot more sense

Reply:
If I change `count_style` to `Optional[str]=None` it causes an error with the `TopButton.update_widget` function...Technically the UserButton doesn't have a `count_style` but nor does it use the `update_widget` function...Not sure how to handle this. Would you change the types that `update_widget` accepts? Or would you change the `TypeAnnotation` for the `count_style` arg?

This was the error message:
`zulipterminal/ui_tools/buttons.py:67: error: Argument 1 to "update_widget" of "TopButton" has incompatible type "Tuple[Optional[str], str]"; expected "Tuple[str, str]"`

Reply:
Fixing the Type of `update widget` would do as previously `unread_count` was hard coded.
Re UserButton, it too displays count in yellow ryt? 
Also I just saw that UserButton uses update_widget in a weird way, it uses `(you)` as count text. Maybe it can be changed in another PR.

Reply:
I can't seem to find where UserButton uses update_widget, do you mind sharing the file/line number where you found that?

Reply:
I fixed the `TopButton.update_widget`'s `TypeAnnotation` to be `Tuple[Optional[str], str]`. I also declared `count_style='unread_count'` in all of the non-starred TopButton children's' `super.__init__` functions....except for the user button because I couldn't find where it was used. Shall I go in and add `count_style='unread_count',` to the `UserButton`'s init function as well?